src/main/java/pages/LogInPage.java

```diff
@@ -25,11 +28,17 @@ public void clickSignInButton() {
         driver.findElement(SIGNIN_BUTTON).click();
     }
 
+    @Step("Fill out credentials, log in into the personal area")
     public PersonalAccountPage logIn(String userName, String password) {
         enterUserName(userName);
         clickSignInButton();
         enterPassword(password);
         clickSignInButton();
         return new PersonalAccountPage(driver);
     }
+
+    @Step("Check the log out happened correctly (comparing titles)")
+    public void assertTitleAfterLogOut(String actTitle, String expTitle) {
+        assertEquals(actTitle, expTitle);
```

Comment:
I don't think you need to have this method with assertions here at the page layer

Reply:
fixed